Formula/usbutils.rb

```diff
@@ -0,0 +1,43 @@
+class Usbutils < Formula
+  # Some of the information is based on Macports 'usbutils':
```

Comment:
Can we condense the level of code comments here a bit, please?

IMO the only necessary comments are the lines about why it is pinned on the version it is.

Reply:
Done: cleaned up the comments.